src/System.Management.Automation/engine/regex.cs

```diff
@@ -63,7 +63,7 @@ public sealed class WildcardPattern
         private static readonly Predicate<string> s_matchAll = _ => true;
 
         // wildcard pattern
-        internal string Pattern { get; }
+        internal string Pattern { get; private set; }
```

Comment:
I don't see the setter being used anywhere.

Reply:
Fixed.